src/pages/design/color.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Color
+description: >
+  The Zendesk Garden color system is a set of purposeful colors designed with
+  brand personality, usability, and accessibility in mind. The system is
+  broken down into two core palettes: UI and brand.
+---
+
+## UI colors
+
+The UI colors are used when creating interface elements. Each color has been
+designed to meet accessibility requirements.
+
+### Primary colors
+
+Primary colors are used for the structure of interfaces, actionable items,
+and validation. Colors in the 600, 700, and 800 ranges have a [WCAG
+AA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
+ratio above 4.5:1 against white or 100 backgrounds. The 800 color range
+(excluding yellow) has AAA contrast ratios above 7:1 against white and 100
+backgrounds.
+
+<ColorPalette hues={['grey', 'kale', 'blue', 'green', 'red', 'yellow']} />
+
+### Secondary colors
+
+Secondary colors are used in supplementary UI elements such as icons, tags,
+status badges, and illustrations. Each color has a light and dark hue with a
+default and muted variant. Muted colors are denoted by an "M" prefix, like
+M400.
```

Comment:
code formatting?

Reply:
While this is being discussed, I wanted to note that the designs called for lower-casing the shade, i.e. `fuschia-m600`. Whereas, here we call out the capital "M" notation. Should we show these as they are in code, that is `fuschia-M600`?